README.md

```diff
@@ -1,2 +1,67 @@
-# ext-superset
+# superset-ext
+
 Meltano Superset utility extension
+
+### example meltano.yml
+
+Note: this example uses a name of `ext_superset` instead of `superset` to avoid conflicts and special handling for the
+existing `superset` plugin.
+
+```yaml
+utilities:
+  - name: ext_superset
+    namespace: superset
+    pip_url: apache-superset==2.0.0 flask==2.0.3 werkzeug==2.0.3 jinja2==3.0.1 wtforms==2.3.3 git+https://github.com/meltano/superset-ext.git@main
+    executable: superset_invoker
+    commands:
+      describe:
+        executable: superset_extension
+        args: describe
+      initialize:
+        executable: superset_extension
+        args: initialize
+      invoke:
+        executable: superset_extension
+        args: invoke
+      create_admin:
+        args: create_admin
+        executable: superset_extension
+      ui:
+        args: invoke run --port 8088 --host 127.0.0.1
+    config:
+      ROW_LIMIT: 42
+      SQLALCHEMY_DATABASE_URI: sqlite:///$MELTANO_PROJECT_ROOT/.meltano/utilities/superset/superset.db
+      SUPERSET_WEBSERVER_PORT: 8088
+      SUPERSET_CONFIG_PATH: $MELTANO_PROJECT_ROOT/superset/superset_config.py
```

Comment:
@pandemicsyn I'm getting `FileNotFoundError: [Errno 2] No such file or directory: '/Users/pnadolny/Documents/Git/meltano_project/sdk_cookie_test/test_123/superset/superset_config.py'`.

I have this meltano.yml:

```yaml
plugins:
  utilities:
  - name: superset
    namespace: superset
    pip_url: apache-superset==2.0.0 flask==2.0.3 werkzeug==2.0.3 jinja2==3.0.1 wtforms==2.3.3 
        git+https://github.com/meltano/superset-ext.git@feat/init-extension-take2
    executable: superset_invoker
    commands:
      describe:
        args: describe
        executable: superset_extension
      initialize:
        args: initialize
        executable: superset_extension
      create_admin:
        args: create_admin
        executable: superset_extension
      ui: run --port 8088 --host 127.0.0.1
    config:
      ROW_LIMIT: 42
      SQLALCHEMY_DATABASE_URI: sqlite:///$MELTANO_PROJECT_ROOT/superset/superset.db
      SUPERSET_WEBSERVER_PORT: 8088
      SUPERSET_CONFIG_PATH: $MELTANO_PROJECT_ROOT/superset/superset_config.py
      SECRET_KEY: OaN2xDksQXXygKmlLINUNxIKMWE3z2WK2gtJEdRaFYkREEiialUo/+Um
      FLASK_APP: superset
```

And I ran:

```bash
meltano install utility superset
meltano invoke superset:initialize
```

Was it supposed to create that file for me or did I miss a step?

Reply:
@pandemicsyn after our zoom chat these issues are no longer relevant. Everything else looked good to me! I got the samples loaded and everything worked, no permissions issues.

I would make one note that the current superset hub instructions have a command `create-admin` with a dash and this one is and underscore.

Reply:
easy-peasy , updated to use `create-admin`